includes/shortcodes/class.llms.shortcode.checkout.php

```diff
@@ -57,7 +57,7 @@ private static function checkout( $atts ) {
 				}
 			}
 			if ( ! $access ) {
-				llms_print_notice( 'You must be a member in order to purchase this access plan.', 'error' );
+				llms_print_notice( __( 'You must be a member in order to purchase this access plan.' ), 'error' );
```

Comment:
The `__()` method requires the additional `$domain` parameter ("lifterlms") otherwise it won't be included in the `.pot` file and will therefore not be translatable.